local/Makefile

```diff
@@ -0,0 +1,30 @@
+all: init-all
+
+init-all: init-db init-proxy init-app set-configs install-apps init-cron
+
+init-db:
+	docker-compose up -d db
+	until docker-compose exec db pg_isready; do echo "Awaiting for Postgres"; sleep 5; done
+
+init-proxy:
+	docker-compose up -d nginx-proxy mkcert
+
+init-app: 
+	docker-compose up -d web app
+	until docker-compose exec --user www-data app php occ status --output=json | grep -e "{\"installed\":true,"; do echo "Awaiting for NextCloud installation"; sleep 10; done
+
+install-apps:
```

Comment:
It looks very specific. This repository is more generic. I suggest removing this part.